ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/basefee-input.js

```diff
@@ -0,0 +1,144 @@
+  }).toNumber();
+};
+
+const multiplyCurrencyValues = (baseFee, value, numberOfDecimals) =>
+  multiplyCurrencies(baseFee, value, {
+    numberOfDecimals,
+    multiplicandBase: 10,
+    multiplierBase: 10,
+  }).toNumber();
+
+const BasefeeInput = () => {
+  const t = useI18nContext();
+  const { gasFeeEstimates, estimateUsed, maxFeePerGas } = useGasFeeContext();
+  const { estimatedBaseFee } = gasFeeEstimates;
+  const {
+    numberOfDecimals: numberOfDecimalsPrimary,
+  } = useUserPreferencedCurrency(PRIMARY);
+  const {
+    currency,
+    numberOfDecimals: numberOfDecimalsFiat,
+  } = useUserPreferencedCurrency(SECONDARY);
+
+  const advancedGasFeeValues = useSelector(getAdvancedGasFeeValues);
+
+  const [editingInGwei, setEditingInGwei] = useState(false);
+
+  const [maxBaseFeeGWEI, setMaxBaseFeeGWEI] = useState(() => {
+    if (
+      estimateUsed !== PRIORITY_LEVELS.CUSTOM &&
+      advancedGasFeeValues?.maxBaseFee
+    ) {
+      return multiplyCurrencyValues(
+        estimatedBaseFee,
+        advancedGasFeeValues.maxBaseFee,
+        numberOfDecimalsPrimary,
+      );
+    }
+    return maxFeePerGas;
+  });
+
+  const [maxBaseFeeMultiplier, setMaxBaseFeeMultiplier] = useState(() => {
+    if (
+      estimateUsed !== PRIORITY_LEVELS.CUSTOM &&
+      advancedGasFeeValues?.maxBaseFee
+    ) {
+      return advancedGasFeeValues.maxBaseFee;
+    }
+    return divideCurrencyValues(maxFeePerGas, estimatedBaseFee);
+  });
+
+  const [, { value: baseFeeInFiat }] = useCurrencyDisplay(
+    decGWEIToHexWEI(maxBaseFeeGWEI),
+    { currency, numberOfDecimalsFiat },
+  );
+
+  const updateBaseFee = useCallback(
+    (value) => {
+      if (editingInGwei) {
+        setMaxBaseFeeGWEI(value);
+        setMaxBaseFeeMultiplier(divideCurrencyValues(value, estimatedBaseFee));
+      } else {
+        setMaxBaseFeeMultiplier(value);
+        setMaxBaseFeeGWEI(
+          multiplyCurrencyValues(
+            estimatedBaseFee,
+            value,
+            numberOfDecimalsPrimary,
+          ),
+        );
+      }
+    },
+    [
+      editingInGwei,
+      estimatedBaseFee,
+      numberOfDecimalsPrimary,
+      setMaxBaseFeeGWEI,
+      setMaxBaseFeeMultiplier,
+    ],
+  );
+
+  return (
+    <FormField
+      onChange={updateBaseFee}
+      titleText={t('maxBaseFee')}
+      titleUnit={editingInGwei ? 'GWEI' : `(${t('multiplier')})`}
+      tooltipText={t('advancedBaseGasFeeToolTip')}
+      titleDetail={
+        <Button
+          className="advanced-gas-fee-input__edit-link"
+          type="link"
+          onClick={() => setEditingInGwei(!editingInGwei)}
+        >
+          <I18nValue
+            messageKey={editingInGwei ? 'editInMultiplier' : 'editInGwei'}
+          />
+        </Button>
+      }
+      value={editingInGwei ? maxBaseFeeGWEI : maxBaseFeeMultiplier}
+      detailText={
+        editingInGwei
+          ? `${maxBaseFeeMultiplier}x ${`≈ ${baseFeeInFiat}`}`
+          : `${maxBaseFeeGWEI} GWEI ${`≈ ${baseFeeInFiat}`}`
```

Comment:
So I spoke with Jake about this detail text a week or so ago and if you revisit the [designs](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW/1559-Gas-Controls---v2.1?node-id=460%3A4673) the first part of each of these (mutilplier/gwei) has been removed.

Reply:
yep actually this code change was started previous to design changes, ui tweaks are already planned in a separate task: https://github.com/MetaMask/metamask-extension/issues/12771